### PR TITLE
feat(plasma-ui): add labels for time pickers

### PR DESCRIPTION
**Pull request by Yakutoc**
## Release Notes

Добавлены `label` для компонента `TimePicker`.

### What/why Changed

Что бы было понятно с чем пользователь имеет дело было решено добавить label. 

### Стало

Screenshot 2023-09-27 at 19 46 27

Screenshot 2023-09-27 at 19 46 34

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.749.6340893068.ts'>borderRadius_default_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.749.6340893068.ts'>borderRadius_plasma_b2c_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.749.6340893068.ts'>borderRadius_plasma_web_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.749.6340893068.ts'>borderRadius_sberHealth_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.749.6340893068.ts'>borderRadius_sbermarket_business_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.749.6340893068.ts'>borderRadius_sbermarket_metro_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.749.6340893068.ts'>borderRadius_sbermarket_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.749.6340893068.ts'>borderRadius_sbermarket_selgros_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.749.6340893068.ts'>borderRadius_sberonline_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.749.6340893068.ts'>borderRadius_sberprime_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.749.6340893068.ts'>borderRadius_stylesSalute_react-native--canary.749.6340893068.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.749.6340893068.swift'>color_default_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.749.6340893068.kt'>color_default_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.749.6340893068.ts'>color_default_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.749.6340893068.xml'>color_default_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.749.6340893068.swift'>color_plasma_b2c_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.749.6340893068.kt'>color_plasma_b2c_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.749.6340893068.ts'>color_plasma_b2c_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.749.6340893068.xml'>color_plasma_b2c_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.749.6340893068.swift'>color_plasma_web_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.749.6340893068.kt'>color_plasma_web_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.749.6340893068.ts'>color_plasma_web_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.749.6340893068.xml'>color_plasma_web_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.749.6340893068.swift'>color_sberHealth_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.749.6340893068.kt'>color_sberHealth_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.749.6340893068.ts'>color_sberHealth_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.749.6340893068.xml'>color_sberHealth_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.749.6340893068.swift'>color_sbermarket_business_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.749.6340893068.kt'>color_sbermarket_business_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.749.6340893068.ts'>color_sbermarket_business_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.749.6340893068.xml'>color_sbermarket_business_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.749.6340893068.swift'>color_sbermarket_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.749.6340893068.kt'>color_sbermarket_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.749.6340893068.swift'>color_sbermarket_metro_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.749.6340893068.kt'>color_sbermarket_metro_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.749.6340893068.ts'>color_sbermarket_metro_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.749.6340893068.xml'>color_sbermarket_metro_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.749.6340893068.ts'>color_sbermarket_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.749.6340893068.swift'>color_sbermarket_selgros_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.749.6340893068.kt'>color_sbermarket_selgros_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.749.6340893068.ts'>color_sbermarket_selgros_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.749.6340893068.xml'>color_sbermarket_selgros_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.749.6340893068.swift'>color_sbermarket_wlbusiness_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.749.6340893068.kt'>color_sbermarket_wlbusiness_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts'>color_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.749.6340893068.xml'>color_sbermarket_wlbusiness_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.749.6340893068.xml'>color_sbermarket_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.749.6340893068.swift'>color_sberonline_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.749.6340893068.kt'>color_sberonline_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.749.6340893068.ts'>color_sberonline_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.749.6340893068.xml'>color_sberonline_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.749.6340893068.swift'>color_sberprime_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.749.6340893068.kt'>color_sberprime_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.749.6340893068.ts'>color_sberprime_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.749.6340893068.xml'>color_sberprime_xml--canary.749.6340893068.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.749.6340893068.swift'>color_stylesSalute_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.749.6340893068.kt'>color_stylesSalute_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.749.6340893068.ts'>color_stylesSalute_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.749.6340893068.xml'>color_stylesSalute_xml--canary.749.6340893068.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.749.6340893068.ts'>shadow_default_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.749.6340893068.ts'>shadow_plasma_b2c_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.749.6340893068.ts'>shadow_plasma_web_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.749.6340893068.ts'>shadow_sberHealth_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.749.6340893068.ts'>shadow_sbermarket_business_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.749.6340893068.ts'>shadow_sbermarket_metro_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.749.6340893068.ts'>shadow_sbermarket_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.749.6340893068.ts'>shadow_sbermarket_selgros_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts'>shadow_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.749.6340893068.ts'>shadow_sberonline_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.749.6340893068.ts'>shadow_sberprime_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.749.6340893068.ts'>shadow_stylesSalute_react-native--canary.749.6340893068.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.749.6340893068.ts'>spacing_default_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.749.6340893068.ts'>spacing_plasma_b2c_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.749.6340893068.ts'>spacing_plasma_web_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.749.6340893068.ts'>spacing_sberHealth_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.749.6340893068.ts'>spacing_sbermarket_business_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.749.6340893068.ts'>spacing_sbermarket_metro_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.749.6340893068.ts'>spacing_sbermarket_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.749.6340893068.ts'>spacing_sbermarket_selgros_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts'>spacing_sbermarket_wlbusiness_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.749.6340893068.ts'>spacing_sberonline_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.749.6340893068.ts'>spacing_sberprime_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.749.6340893068.ts'>spacing_stylesSalute_react-native--canary.749.6340893068.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.749.6340893068.swift'>typo_mage_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.749.6340893068.kt'>typo_mage_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.749.6340893068.ts'>typo_mage_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.749.6340893068.swift'>typo_plasma_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.749.6340893068.kt'>typo_plasma_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.749.6340893068.ts'>typo_plasma_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.749.6340893068.swift'>typo_ruler_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.749.6340893068.kt'>typo_ruler_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.749.6340893068.ts'>typo_ruler_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.749.6340893068.swift'>typo_sage_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.749.6340893068.kt'>typo_sage_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.749.6340893068.ts'>typo_sage_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.749.6340893068.swift'>typo_sbermarket_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.749.6340893068.kt'>typo_sbermarket_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.749.6340893068.ts'>typo_sbermarket_react-native--canary.749.6340893068.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.749.6340893068.swift'>typo_soulmate_ios-swift--canary.749.6340893068.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.749.6340893068.kt'>typo_soulmate_kotlin--canary.749.6340893068.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.749.6340893068.ts'>typo_soulmate_react-native--canary.749.6340893068.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.183.0-canary.749.6340893068.0
  npm install @salutejs/plasma-ui@1.215.0-canary.749.6340893068.0
  # or 
  yarn add @salutejs/plasma-temple@1.183.0-canary.749.6340893068.0
  yarn add @salutejs/plasma-ui@1.215.0-canary.749.6340893068.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-749/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-749/
ui storybook: http://plasma.sberdevices.ru/pr/pr-749/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-749/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-749/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-749/
ui storybook: http://plasma.sberdevices.ru/pr/pr-749/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-749/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-749/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-749/
ui storybook: http://plasma.sberdevices.ru/pr/pr-749/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-749/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-749/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-749/
ui storybook: http://plasma.sberdevices.ru/pr/pr-749/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-749/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
+    &[data-label]::before {
+        content: attr(data-label);
+        position: absolute;
+        left: 0;
+        margin-top: ${({ controls }) => (controls ? '-2.5rem' : '-1.5rem')};
+        width: 100%;
+        color: ${whiteTertiary};
+        ${footnote1};
+        font-weight: normal;
+        font-size: ${({ size = DEFAULT_PICKER_SIZE }) => labelFontSize[size]};
```
[the remark this answers is not in the document]
Comment (Yeti-or) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:38]: кажется это не кореектно в этом месте

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx [hunk 2]
```diff
@@ -232,11 +283,12 @@ export const TimePicker = ({
                     onChange={onHoursChange}
                     aria-label={hoursAriaLabel}
                     disableScrollSnapAlign={disableScrollSnapAlign}
+                    data-label={hoursLabel || null}
```
Comment (Yeti-or) [inline packages/plasma-ui/src/components/Pickers/TimePicker.tsx:286]: а если  undefined передавать плохо будет? нужен именно `null` ?

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx [hunk 3]
```diff
@@ -1,18 +1,51 @@
...
+    xs: '0.625rem',
+};
+
+const StyledSimpleTimePicker = styled(SimpleTimePicker)`
+    &[data-label] {
+        margin-top: 2rem;
+    }
+
+    &[data-label]::before {
+        content: attr(data-label);
```
Comment (Yeti-or) [inline packages/plasma-ui/src/components/Pickers/TimePicker.tsx:30]: 🦖  мне нравится

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx [hunk 4]
```diff
@@ -1,18 +1,51 @@
...
+        position: absolute;
+        left: 0;
+        margin-top: ${({ controls }) => (controls ? '-2.5rem' : '-1.5rem')};
+        width: 100%;
+        color: ${whiteTertiary};
+        ${footnote1};
+        font-weight: normal;
```
Comment (Yeti-or) [inline packages/plasma-ui/src/components/Pickers/TimePicker.tsx:37]: у foontone не задано?

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:38]: @Yeti-or Ты про то что он кинул алерт?

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 2 above)
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:286]: @Yeti-or В принцпе можно и undefined. поменяю

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 4 above)
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:37]: @Yeti-or Это что бы было и в новой и старой типографики одинаково

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 3 above)
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:30]: @Yeti-or А почему эмоджи динозавра? )

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 2 above)
Comment (Yeti-or) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:286]: ну я имел ввиду что-то такое:
`data-label={hoursLabel}`

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 4 above)
Comment (Yeti-or) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:37]: 👌

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 2 above)
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:286]: @Yeti-or А ты про это, если пустая строка то атрибут появиться на теге.

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:38]: @Yeti-or 

Нам точно править не нужно. Скорее нужно скорректировать правила.

[the remark this answers is not in the document]
Comment (Yeti-or) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:38]: А ты понял где прочитать пример эксплоита?

[the remark this answers is not in the document]
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:38]: @Yeti-or 

https://github.com/eslint-community/eslint-plugin-security/blob/main/docs/rules/detect-object-injection.md

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-749/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-749/
ui storybook: http://plasma.sberdevices.ru/pr/pr-749/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-749/temple-storybook/

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx [hunk 5]
```diff
@@ -66,6 +99,21 @@ export interface TimePickerProps extends Omit<SimpleTimePickerProps, 'type' | 'r
...
+
+    /**
+     * Label для picker "минуты"
+     */
+    minutesLabel?: string;
+
+    /**
+     * Label для picker "секунды"
+     */
+    secondsLabel?: string;
```
Comment (neretin-trike) [inline packages/plasma-ui/src/components/Pickers/TimePicker.tsx:116]: а мы не хотим значения по умолчанию делать и дать флаг типа "showLabels"?

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-ui/src/components/Pickers/TimePicker.tsx (hunk 5 above)
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/TimePicker.tsx:116]: @neretin-trike Еще один проп плюсом к этим трем? 

В моей голове это было как вот им проп пусть что хотят туда и пишут.

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-749/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-749/
ui storybook: http://plasma.sberdevices.ru/pr/pr-749/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-749/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by kayman233 — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-749/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-749/
ui storybook: http://plasma.sberdevices.ru/pr/pr-749/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-749/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.173.0-dev.0, @salutejs/plasma-temple@1.183.0-dev.0, @salutejs/plasma-ui-docs@0.222.0-dev.0, @salutejs/plasma-ui@1.215.0-dev.0 🚀

**Pull request merged**